px-context-browser-filter.html

```diff
@@ -13,7 +14,7 @@
       <input
           id="filter"
           class="text-input filter__box"
-          placeholder="Filter"
+          placeholder="{{localize('Filter')}}"
```

Comment:
See comment below, if you accept then this would become `placeholder="[[placeholder]]"`